.github/workflows/nightly-master-e2e.yaml

```
@@ -43,9 +43,22 @@ jobs:
       matrix:
         include: ${{ fromJSON(needs.setup-matrix.outputs.matrix) }}
     steps:
+      # Initial checkout without submodules so that the local bitwarden action is available.
       - name: Checkout repository
         uses: actions/checkout@v7
 
+      - name: Loading Secrets
+        uses: ./.github/actions/bitwarden
```

actionlint (1.7.12)

[error] description is required in metadata of "Bitwarden Credentials Integration" action at "/home/jailuser/git/.github/actions/bitwarden/action.yaml"

(action)